CONTRIBUTING.md

```diff
@@ -137,6 +137,10 @@ Try to run `make -B build/ihp-lib` to create the symlink.
 
 In the project directory, try `make -B .envrc`
 
+### `*** WARNING: . is writable by someone else, IGNORING! Suggested fix: execute 'chmod go-w .'`
```

Comment:
The initial error was `error: Could not find moduleGenerated.Types'`, maybe we could use that one here. The chmod error only came up after debugging :)

Reply:
True that. Just updated the title and pushed